samples/web/content/apprtc/apprtc.py

```diff
@@ -7,10 +7,14 @@
 This module demonstrates the WebRTC API by implementing a simple video chat app.
 """
 
+# Enables loading third_party modules
```

Comment:
style nit: can we order this as follows:

system imports [os, sys, random, threading etc]
third party imports [sys.path.append..., google.appengine.api etc]
app imports [analytics, constants etc]